relm4-components/src/alert/mod.rs

```diff
@@ -0,0 +1,244 @@
+//! Reusable and easily configurable alert component.
+//!
+//! **[Example implementation](https://github.com/AaronErhardt/relm4/blob/main/relm4-examples/examples/alert.rs)**
+
+use gtk::prelude::{BoxExt, ButtonExt, GtkWindowExt, OrientableExt, WidgetExt};
+use relm4::{gtk, Component, ComponentParts, ComponentSender, RelmWidgetExt};
+
+static LIBADWAITA_ENABLED: bool = cfg!(feature = "libadwaita");
+static COMPONENT_CSS: &str = include_str!("style.css");
+
+static ERROR_CSS: &str = "error";
+static TITLE_CSS: &str = "title-2";
+static DESTRUCTIVE_CSS: &str = "destructive-action";
+static FLAT_CSS: &str = "flat";
```

Comment:
Shouldn't is be `const` instead of `static`?

Reply:
I chose `static`, as otherwise the string gets inlined everywhere it's used. Would you rather it not behave like that?

After #650 gets merged, these shouldn't be required anyway though. If you'd prefer things to be `const` I'll need to get things changed up before that PR gets merged as well.

Reply:
I prefer `const` when possible. I don't think rustc will inline `const` items everywhere, rather `const` items are not guaranteed to have a certain memory location. Since we don't need `static` in the sense of a global variable here and rather define constants, `const` should be idiomatic (and maybe even slightly more performant).

Reply:
[It does look like Rust inlines `const` usage](https://doc.rust-lang.org/std/keyword.const.html):

> ...constants are inlined wherever they’re used, making using them identical to simply replacing the name of the `const` with its value.

I'm not sure if that's a problem for you, but I thought I'd mention it just in case. If that's fine, I'm good to move them over to using `const`.

Reply:
They might be inlined, but the compiler will not create unnecessary space for them. In fact, I just checked out of curiosity and the compiler will emit the exact same assembly regardless of `static` or `const` being used, with the only difference being the name of the memory region it creates for the constant value. So since `const` is semantically more correct, I'd simply use that.

Reply:
I wasn't aware of that, I guess the docs misled me a bit :p. Anyhow, I can't argue with what's proven, so I'll go ahead and get everything moved over.